README.md

```diff
@@ -41,142 +23,74 @@ or if you use `yarn`
 yarn add did-jwt
 ```
 
-## API
-
-### Creating a JWT
-
-Use the `createJWT()` function
+## Example
 
+### 1. Create a did-JWT
 ```js
-import { createJWT, SimpleSigner } from 'did-jwt'
-
-const signer = SimpleSigner('PRIVATEKEY')
+const didJWT = require('did-jwt')
+const signer = didJWT.SimpleSigner('fa09a3ff0d486be2eb69545c393e2cf47cb53feb44a3550199346bdfa6f53245');
```

Comment:
Maybe add a comment that this is just an example of a private key?

Reply:
It's an actual identity private key so readers can cut/paste the example and run it.

How can we do this and not confuse the reader?

Reply:
I added some text that clarifies the signer key bit.  LMK what you think, and if this is a good idea or should we do something different?